.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
```suggestion
    "source.fixAll.eslint": true
```

Reply:
@PeerRich it's actually due to this new vs code update https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own